producer.go

```diff
@@ -99,7 +98,13 @@ func (p *Producer) consumeDeliveries() {
 			}
 		} else {
 			p.log.Printf("Error consuming delivery event: Unknown event type (%s)", ev)
-			atomic.AddUint64(&stats.producerErr, 1)
+
+			// Ignore 'connection reset by peer' errors since they manifest because
+			// of the connection reaper.
```

Comment:
Should we also mention explicitly that these are non-errors? The "since they manifest because of the connection reaper" part doesn't explain much.

Reply:
But they are errors, the are connection errors, tcp errors. It is just that they are ignorable by us in this context.

Reply:
Let me rephrase: should we elaborate more on why we treat them as non-errors (as opposed to merely referring to the connection reaper, which doesn't provide much context)?